lib/schlepp/format/base.rb

```diff
@@ -52,15 +53,28 @@ def after(&block)
       # throws an error if a required file is not found. runs our before,
       # then process_file, then our after.
       def process!
-        data = read
-        if data.nil?
-          raise "Required file not found: #{File.join(Format.cwd, name)}" if required
-          return
+       files = retrieve_file_list
+        @before.call if @before
+        files.each do |file|
+          data = read(file)
+          if data.nil?
+            raise "Required file not found: #{File.join(Format.cwd, name)}" if required
+            return
+          end
+
+          result = process_file(data)
         end
+        @after.call if @after
+      end
 
-        @before.call(data) if @before
-        result = process_file(data)
-        @after.call(result) if @after
+      # Glob support to pull file list for processing.
+      def retrieve_file_list
+        if name.is_a?(Hash)
+          return nil unless name[:glob]
+          Dir.glob(name[:glob])
```

Comment:
What do you think about just having name be an array, and if it's an array, then we glob each of them?